enclave-runtime/src/lib.rs

```diff
@@ -69,11 +69,11 @@ use substrate_api_client::{compose_extrinsic_offline, ExtrinsicParams};
 
 mod attestation;
 mod empty_impls;
-mod extrinsic_daemon;
 mod global_components;
 mod initialization;
 mod ipfs;
 mod ocall;
+mod stf_task_thread;
```

Comment:
can we move `stf_task_thread` to `stf-task-handler` crate?

Reply:
This is the problem I try to recall in our discussion. There is recursive include from enclave-runtime to litentry/* , then back to enclave-runtime.

Reply:
Which one exactly?

Reply:
Currently, inside `enclave-runtime/Cargo.toml` it depends on `lc-stf-task-handler` and `lc-identity-verify-handler`. `stf_task_thread` also depends on the `error` inside `enclave-runtime`. If I move `stf_task_thread` into `litentry/core` folder, recursive include happened.

Reply:
I see, thanks. Let's then keep `stf_task_thread` in `enclave-runtime` but have a separate `stf_task_receiver.rs` under `litentry/core/stf-task-handler`. It should only contain the loop and doesn't need to refer to the `Error` type